mas-foundation/src/main/java/com/ca/mas/core/conf/ConfigurationManager.java

```diff
@@ -40,12 +40,16 @@
 public class ConfigurationManager {
 
     private static final String CONNECTED_GATEWAY_CONFIG = "connected_gateway.json";
+
     private ConfigurationProvider connectedGatewayConfigurationProvider = null;
     private Context appContext;
     private List<Config> appConfigs;
     private String configurationFileName = null;
     private boolean enablePKCE = true;
     private boolean idTokenValidation = true;
+    private String jwks;
```

Comment:
I would prefer not to put the jwks cache in ConfigurationManager, I would suggest to put it as a static field under JWTRS256Validator. 
Not only memory cache, but we also need to cache the content in SharePreference (by gateway), so that we don't hit those endpoints everytime we re-open the App.